pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -528,6 +528,11 @@ spec:
             ports:
             - containerPort: 3260
               protocol: TCP
+            env:
```

Comment:
what if user not given these variables?

Reply:
then its invalid value and cstor-istgt will  take default values